O3-1.tex

```diff
@@ -268,7 +268,7 @@ \section{The LSST software stack}
 access and orchestration middleware, as well as the database and user
 interface components.
 
-Algorithm development for the LSST software builds on the expertize
+Algorithm development for the LSST software is building on the expertize
```

Comment:
Both spellings are valid, but they mean different things: "expertise" is a noun, "expertize" is a verb. "Expertise" is correct here.

(Mind you, what do I know with my British English...?!)